src/nspawn/nspawn.c

```diff
@@ -1395,6 +1395,12 @@ static int copy_devnodes(const char *dest) {
 
                 } else {
                         if (mknod(to, st.st_mode, st.st_rdev) < 0) {
+                                /*
+                                 * This is some sort of protection too against
+                                 * recursive userns chown on shared /dev/
```

Comment:
@keszybz that comment is to warn future modification on this part of code in case private users is implied, right now the code does a recursive chown to shift uids/gids... so if you pre-mount that and pass it to nspawn with root privileges you may chown /dev/* nodes and shift their uids/gids... which will have impacts on other entities that are using the same nodes... So that comment is a warning to developers.

Now for log_notice I don't get you ? it's only emitted on errno == EEXIST, and for the order yes it depends, I really don't feel that strong about it... but will make the change. Thanks